lib/bamboo/email.ex

```diff
@@ -164,6 +183,7 @@ defmodule Bamboo.Email do
 
       put_private(email, :tags, "welcome-email")
   """
+  @spec put_private(__MODULE__.t, atom, String.t) :: __MODULE__.t
```

Comment:
Rather than `String.t` it should be `any` because you can put whatever you like in the private key